rre-dataset-generator/src/search_engine/data_store.py

```diff
@@ -112,3 +125,77 @@ def has_rating_score(self, query_id: str, doc_id: str) -> bool:
         context = self._get_query_rating_context_by_id(query_id)
         return context.has_rating_score(doc_id)
 
+    
+    @staticmethod
+    def query_context_docs_to_dict(query: QueryRatingContext, documents: List[Document]) -> dict:
+        return {
+            "query_id": query.get_query_id(),
+            "query_text": query.get_query(),
+            "doc_ids": query.get_doc_ids(),
```

Comment:
`query.get_doc_ids()` returns the list of keys of `query._doc_id_to_rating_score`.
Why do we need to keep both `doc_ids` and `doc_ratings` if `doc_ratings` is the superset of `doc_ids`?